docs/core/toc.yml

```diff
@@ -238,16 +238,17 @@
   items:
   - name: Overview
     href: tools/index.md
-  - name: Tools
+  - name: Global and local tools
```

Comment:
Wouldn't this just be .NET tools but then we could add displayname for global tools and local tools?

Reply:
I was thinking that "tools" is still overloaded somewhat by itself, and "global and local" makes it clearer what kind of tools the section is about.  I'm open to changing it if you think just "tools" by itself is preferable here.